lib/aca_entities/families/family_member_reference.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module Families
+    class FamilyMemberReference < Dry::Struct
+      # TODO: Person hbx_id until we add hbx_id on family member
```

Comment:
EA assigns hbx_id to FamilyMember class.  That is value to include here

Reply:
Removed comment, hbx_id filed need to added to enroll family member modal.